### PR TITLE
Transfer OpenTelemetry OTLP docs to public site

### PR DESCRIPTION
**Pull request by rhetoric101**
These are the docs that were created on the private site and are now ready for review. Please look closely at the following:

INTRODUCTION

* Added links in "Benefits of OpenTelemetry"
* Added Slack link in "Should I use OpenTelemetry instrumentation or New Relic agents?"
* Added new wording and tables to "How OpenTelemetry works with New Relic," including a link to Slack.

QUICK START

Step 3 was revamped and divided into two sub-sections. Please review the entire step and links.

BEST PRACTICES

* Added a new mini table of contents at top.
* Added two new sub-sections to the Traces section.
* Added new Metrics section. 
* Added new introductory content to existing Log section.

VIEW YOUR OPENTELEMETRY DATA

* Added details to each view section explaining how to ensure the UI lights up with your data.
* Added new section "Narrow down your data with filters"
* Modified the first paragraph in "Distributed tracing"
* Metrics explorer  now has a screenshot

FEDRAMP-COMPLIANT ENDPOINTS

This document now has the FedRamp OTLP endpoints and is references from the quick start: https://docswebsitedevelop-rhsotlpgadoc7276.gtsb.io/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#otlp-api

I didn't make any changes to "Legacy New Relic OpenTelemetry Exporters" but you are welcome to review it.

### CODE REVIEW

**Comment by rhetoric101** [comment]
@mmfred, this work is covered in DOC-7352, which is a follow up to DOC-7276.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx
Comment (rkbodenner) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:19]:
```suggestion
### Narrow down your data with filters [#filter]
```

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx
Comment (mmfred) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:30]: What do you see as the relationship between the steps and this info - I almost want a heading 3 or something to transition between the steps and the paragraphs of info. It might just be that I'm not used to a bunch of paragraphs after a step though. Or, it might be that indenting these paragraphs to indicate they're the follow on to the steps is what's most ideal.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx
Comment (mmfred) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:207]: Is this your TO DO?

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx
Comment (mmfred) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:274]: improvement - to make this flow in the list, change to:

The `trace.id` and `span.id` attributes, to correlate with trade data

**Review by mmfred — APPROVED** [review]
Hi Rob - many of these are nice to have changes, rather than required. But there are a few broken links and dropped words. I'm approving so you can fix those things without having to worry or wait on a review.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx:163]: @mmfred , good catch!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx:181]: @mmfred, if you're asking if we need the word "dimensions" in that sentence, we do need that word.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx:183]: @mmfred I'm glad you found that! I'm fixing this, now!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx:212]: @mmfred, this means that if OpenTelemetry makes a code change that breaks everyone's instances, we will respond and make sure we make any necessary changes on our end. This was the language from the SME.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx:194]: @mmfred, I wish we could control the width of tables, but alas, I don't think we can. Originally, these tables had description columns, but we took them out to avoid redundancy with another topic. Let's address this later with a CSS fix.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx:179]: @mmfred, nice suggestion! I added this link!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start.mdx:82]: @mmfred, these are endpoints where users send their data instead of browser links, so we're good!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start.mdx:134]: @mmfred, that is helpful feedback. We need to clarify the collector option, but we'll do that later.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:24]: @mmfred, nice catch!!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:39]: @mmfred, good feedback. We'll have to address this later.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:74]: @mmfred, coincidentally, the SME gave me some other fixes to this area and I added those commas following those sentences!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:78]: @mmfred Sounds like you're suggesting we drop the word "field." Not sure at this point.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:87]: @mmfred great suggestion! I added an anchor!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (mmfred) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:78]: No, I was doing a subject verb agreement check. As is, the sentence should have an "is" instead of an "are" - so it would be: When `traceId` or `spanId` is not present, the span will be dropped.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:104]: @mmfred, I added a verb!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:58]: @mmfred, we'll fix this later. Thanks for the heads-up!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:125]: @mmfred: Passive=gone!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:223]: @mmfred, very helpful! Fixed it!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:334]: @mmfred, I caught this when adding other new content. Thanks!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:30]: @mmfred, I switched this up based on SME feedback. So, we now have two sub-sections.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:207]: @mmfred Now, that was embarrassing! I'm removing that!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:274]: @mmfred , good tip, but I'm going to hold off for the moment.

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:309]: @mmfred good catch! We fixed it!!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts.mdx:37]: @mmfred, we're not going to address this at the moment. Good clarification, though!

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx
Comment (rhetoric101) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx:19]: @rkbodenner, I revamped this page, separating your data filtering information from the entity filtering information that was in the original. This anchor was changed in the process.

**Pull request merged**
